content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md

```diff
@@ -0,0 +1,102 @@
+
+This article provides an overview of Chainguard Enforce and its components.
+
+
+## What is Chainguard Enforce?
+
+Chainguard Enforce is a developer platform for end-to-end software supply chain security that operates at the level of Kubernetes fleets. Once you install Enforce onto a Kubernetes cluster and apply a policy, Chainguard Enforce will immediately begin analyzing metadata associated with images running in the cluster to determine whether it satisfies the requirements defined within the policy. Specifically, it inspects the given cluster's [software artifacts](https://console.enforce.dev/policies/catalog), such as an image's software bill of materials (SBOM). Based on the details within the images' metadata, Chainguard Enforce will determine whether those images satisfy the policy.
+
+Chainguard Enforce provides a rich IAM model similar to the likes of AWS and GCP, allowing you to organize policies and clusters into a hierarchy of groups through its Identity and Access Management (IAM) model. When you install Enforce onto a cluster, you associate that cluster with an existing IAM group. Likewise, when you create a policy, you associate it with a group and it is automatically applied to any clusters already associated with that group. 
+
+One feature that makes Chainguard Enforce unique is continuous verification. This means that Enforce will scan the cluster on a regular cadence to ensure that it complies with any policies you've applied to it. Chainguard Enforce uses an [admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/) to block any requests that violate a cluster's policy. 
+
+Typically, admission controllers only check policies when Kubernetes resources are being created. If new policies are rolled out, or the state of an application’s compliance degrades over time, then admission controllers will not catch the compliance degradation. Chainguard Enforce, however, continuously scans the cluster against all of the latest policies. This means that if a deployed application violates a new policy or falls out of compliance, it will catch the policy violation before Kubernetes acts on the request. 
+
+Violations can be sent as Cloud Events, so you can respond in the most appropriate way for your policies and applications. This might be to file a JIRA ticket, page the on-call, delete pods, or trigger a CI/CD pipeline to redeploy the application. Regardless of how you configure your response, the violation will be logged for future reference.
+
+> Note: For more information on how continuous verification works, check out our conceptual article on [Understanding Continuous Verification](../understanding-continuous-verification/).
+
+The following diagram outlines Chainguard Enforce's various components and how they work together. 
+
+![Diagram outlining Chainguard Enforce's architecture.](enforce-diagram.png)
```

Comment:
On the dark background the image is hard to read:

<img width="995" alt="Screenshot 2022-12-15 at 4 48 47 PM" src="https://user-images.githubusercontent.com/5977693/207974441-1ba32dad-28b2-4eda-8586-b92171ae98f0.png">

Reply:
I added a white background and bumped up the font size across the board.